.travis.yml

```diff
@@ -9,12 +9,13 @@ before-install:
   
 script:
   - docker build --target test -t test .
-  - docker run -e TRELLO_KEY  -e TRELLO_TOKEN -e TRELLO_BOARD -e SECRET_KEY -e FLASK_APP='todo_app/app' -e FLASK_ENV='development' -e FLASK_SKIP_DOTENV='True' -e MONGO_USERNAME -e MONGO_PASS  -e MONGO_HOST -e DEFAULT_DATABASE -e LOGIN_DISABLED="True" test
+  - docker run -e TRELLO_KEY  -e TRELLO_TOKEN -e TRELLO_BOARD -e SECRET_KEY -e FLASK_APP='todo_app/app' -e FLASK_ENV='development' -e FLASK_SKIP_DOTENV='True' -e MONGO_CONNECTION_STRING -e LOGIN_DISABLED="True" test
  
 after_success:
   - docker build --target prod -t $DOCKER_USERNAME/todo-app:latest .
   - docker tag $DOCKER_USERNAME/todo-app:latest registry.heroku.com/todo-app-m8/web
   - docker push $DOCKER_USERNAME/todo-app:latest
+  - curl -dH -X POST "https://\$capita2-jorgeferrer-webapp:f7zzpw4hFAtWsnwzvitdYdcesWLD0JMNQZSZYPrlk19qSxRLHj7GRv6ivPHq@capita2-jorgeferrer-webapp.scm.azurewebsites.net/docker/hook"
```

Comment:
Note that:
* this value was obscured in the Azure Portal because it contains a deployment password, so we don't want it committed! We should extract this URL into an encrypted environment variable rather than committing it to our Git Repo
Since this has already been exposed, we should also rotate the secret in Azure; you can do this by hitting "Reset publish profile" from the app's overview, or through "Manage publish profile" on the Deployment Center
* This step is deploying on every push, not just on the branches that we want to deploy! We should switch this step into your `deploy` stage, and we can now remove the Heroku deployments if we want

Reply:
both DB and Web app keys have been rotated.  Webhook url and db connection hidden in a  hidden variable.